### PR TITLE
#41 - Don't run Doclint for javadoc for JDK 8

### PR DESCRIPTION
**Pull request by jccarrillo**
Circumvents issue #41 by not running the doclint for javadoc when building with jdk8

### CODE REVIEW

**Comment by jccarrillo** [comment]
@bengunter, @vankeisb, @lmeadors you have modified this pom.xml, please have a look at this patch as this will allow users to build stripes with jdk 8.

**Comment by vankeisb** [comment]
Hi,
I did that change locally but I think it breaks the 1.7 build, which we use for releasing.
I left it as is since it's a "good thing",or at least it has its advantage : it prevents to ship a 1.8 compiled jar...
So I'm OK with the change but it has to build (release profile) with 1.7, and it'd be nice to fail the release build if java > 1.7.

**Comment by jccarrillo** [comment]
@vankeisb, do you use the "maven release plugin" to release a build?

**Comment by vankeisb** [comment]
Not sure. I don't have the code right here but we inherit a parent pom from Sonatype, which allows us to release.

You can try it out in 1.7 by activating the "release" profile.

File: pom.xml [hunk 1]
```diff
@@ -296,5 +297,25 @@
...
+            <properties>
+                <javadoc.opts>-Xdoclint:none</javadoc.opts>
+            </properties>
+          <build>
+            <plugins>
+              <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
+                <configuration>
+                  <skip>true</skip>
```
Comment (jccarrillo) [inline pom.xml]: @vankeisb, this prevents the source from being built when running in JDK 1.8 or higher.

I tried it by using the `mvn clean install -Psonatype-oss-release` command line.

**Comment by jccarrillo** [comment]
I think the last commit does the trick you wanted. I will squash before any merge. Waiting for any comments.

**Comment by vankeisb** [comment]
Why skip gpg ?

**Comment by jccarrillo** [comment]
I skipped it based on jdk1.8 and the build failed as before. It will prevent shipping with a 1.8 build.

**Comment by jccarrillo** [comment]
@vankeisb thanks for accepting the patch

**Comment by vankeisb** [comment]
My pleasure !

**Pull request merged**